README.md

```diff
@@ -0,0 +1,269 @@
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Portfolio finish mobile version] <a name="about-project"></a>
+
+**[Portfolio finish mobile version]** is only the first step to get an amazing portfolio. It shows a mobile version of my portfolio, in the future I'll add features, animations, and much more.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Version Control System</summary>
+  <ul>
+    <li><a href="https://git-scm.com/">Git</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Cloud Repository</summary>
+  <ul>
+    <li><a href="https://github.com/">Github</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Linters</summary>
+  <ul>
+    <li><a href="https://developer.chrome.com/docs/lighthouse/overview/">Lighthouse</a></li>
+    <li><a href="https://webhint.io/">Webhint</a></li>
+    <li><a href="https://stylelint.io//">Stylelint</a></li>
+</ul>
+</details>
+
+<details>
+  <summary>Layout</summary>
+  <ul>
+    <li><a href="https://github.com](https://developer.mozilla.org/en-US/docs/Glossary/HTML5/">HTML</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Style</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/es/docs/Web/CSS/">CSS</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Applying_Linters]**
+- **[Applying_Github_Flow]**
+- **[Applying_Professional_Documentation]**
+- **[Applying_HTML_basis]**
+- **[Applying_CSS_Flexbox]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Coming_soon]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- **[Visual_Studio_Code]**
+- **[Live_Server_Extension]**
+- **[Web_Browser]**
+
+<!--
+Example command:
+
+```sh
+ gem install rails
+```
+ -->
+
+### Setup
+
+Clone this repository to your desired folder:
+
+- **[https://github.com/henrycadenaec/PortfolioMobileFirst.git]**
+
+Example commands:
+
+```sh
+  cd my-folder
+  git clone git@github.com:myaccount/my-project.git
+```
+
+Open the project folder in Visual Studio Code:
+
+- **[https://github.com/henrycadenaec/PortfolioMobileFirst.git]**
+
+### Install
+
+- **[No_needed]**
+
+<!--
+Example command:
+
+```sh
+  cd my-project
+  gem install
+```
+--->
+
+### Usage
+
+- **[Open index.html file in VS Code, run via live server extension]**
+
+<!--
+Example command:
+
+```sh
+  rails server
+```
+--->
```

Comment:
Kindly, remove this commented code to make your project look more professional and for best practices. 😊